purchases/src/main/kotlin/com/revenuecat/purchases/customercenter/CustomerCenterConfigData.kt

```diff
@@ -0,0 +1,465 @@
+package com.revenuecat.purchases.customercenter
+
+import com.revenuecat.purchases.paywalls.PaywallColor
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+
+typealias RCColor = PaywallColor
+
+private const val HASH_CODE_MULTIPLIER = 31
+
+@Serializable
+internal class CustomerCenterRoot(
+    @SerialName("customer_center") val customerCenter: CustomerCenterConfigData,
```

Comment:
Hmm since these classes are still public, I would try to keep convention and use kotlin syntax... I think it would look weird in the usage as well... I would prefer to have an intermediate "network model" layer and map those to different models to expose them to the rest of the SDK, but then I feel we would be adding a lot of overhead IMO... Lmk what you think!

Reply:
>I would prefer to have an intermediate "network model" layer and map those to different models to expose them to the rest of the SDK

Oh right, yes, since we are using these network models directly, it's indeed good to keep the naming idiomatic Kotlin. I agree that mapping them to different models is probably not worth it.